src/App.js

```diff
@@ -0,0 +1,14 @@
+import { observer } from 'mobx-react';
+import React from "react";
+import './App.css';
+import Market from './components/Market';
+
+const App = observer((props) => {
+	return (
+		<header className="App-header">
+			{<Market />}
```

Comment:
You don't need the curly brackets around `<Market />`. You only need curly brackets when adding Javascript.